src/components/Card/Card.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, useEffect, useState } from 'react';
+import { getItem } from '../../helpers/api';
+import { PHOTO_URL } from '../../helpers/constants';
+import { IcardState } from '../../helpers/interface';
+import { CardItem } from './components/CardItem/CardItem';
+
+export const Card: FC = () => {
+  const URL = PHOTO_URL
```

Comment:
why do you need this const? do you need to assign PHOTO_URL to some extra variable?